style.css

```diff
@@ -1 +1,25 @@
-style.css
\ No newline at end of file
+.taskContainer,
+.answerContainer{
```

Comment:
`task-container` `answer-container`
Maybe you can use just one class `container`?

Reply:
I wonted show answer separately

Reply:
I understand, but then i should add to answer div that class. Ok, i change this.